### ISSUE TITLE
Error: Expected subtree parent to be a mounted class component.

### ISSUE THREAD

**Issue by navono** [react-paper/react-paper-bindings#29]
I have use `create-react-app` to create a demo app, and added `react-paper-bindings`, `App` component like this:
```
import React, { Component } from "react";
import ReactDOM from "react-dom";
import { View, Layer, PointText, Circle } from "react-paper-bindings";

import "./styles.css";

class App extends Component {
  constructor(props) {
    super(props);

    this.state = {
      mounted: false
    };
    this._box = null;
  }

  componentDidMount() {
    this.setState({ mounted: true });
  }

  render() {
    const { mounted } = this.state;
    const box = this._box && this._box.getBoundingClientRect();

    const viewProps = {
      width: (box && box.width) || 1000,
      height: (box && box.height) || 1000
    };

    return (
      <div className="App" ref={ref => (this._box = ref)}>
        {mounted && (
          <View {...viewProps}>
            <Layer>
              <PointText
                point={[200, 200]}
                content={"Click Me"}
                fillColor={"#000000"}
                fontSize={18}
              />
              <Circle center={[80, 50]} radius={30} strokeColor={"black"} />
            </Layer>
          </View>
        )}
      </div>
    );
  }
}

const rootElement = document.getElementById("root");
ReactDOM.render(<App />, rootElement);
```

When start the demo, it's always generate error:
>`Expected subtree parent to be a mounted class component. This error is likely caused by a bug in React. Please file an issue.` 

package.js
```
{
  "name": "paper-demo",
  "version": "0.1.0",
  "private": true,
  "main": "src/index.js",
  "dependencies": {
    "lodash": "^4.17.11",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-paper-bindings": "^1.1.1",
    "react-scripts": "1.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
```

And I have tried the same code in [codesandbox](https://codesandbox.io/s/k1m4p0p5q7), it's worked. Did I miss something or do something wrong?

**Comment by navono** [comment]
as mentioned @gaearon in [issue](https://github.com/facebook/react/issues/13742#issuecomment-425082590).

**Comment by HriBB** [comment]
@navono I have released version [2.0.0](https://github.com/react-paper/react-paper-bindings/releases/tag/v2.0.0) and this problem should be fixed. Please reopen if you still have a problem.
